### PR TITLE
Added timeout as cli arg

### PR DESCRIPTION
**Pull request by freaker2k7**
Sometimes you need more :)

### CODE REVIEW

**Review by OmarM9 — APPROVED** [review]

File: panoramix/decompiler.py [hunk 1]
```diff
@@ -155,7 +155,12 @@ def _decompile_with_loader(loader, only_func_name=None) -> Decompilation:
             if target > 1 and loader.lines[target][1] == "jumpdest":
                 target += 1
 
-            @timeout_decorator.timeout(60 * 3, timeout_exception=TimeoutInterrupt)
```
Comment (Carsonmarvel1) [inline panoramix/decompiler.py:158]: Timestamp

**Review by Carsonmarvel1 — APPROVED** [review]
Return coin

File: panoramix/decompiler.py (hunk 1 above)
Comment (Carsonmarvel1) [reply panoramix/decompiler.py:158]: Thats why you take my coin

**Review by Anastasia1510 — APPROVED** [review]

**Review by axljoker03 — APPROVED** [review]

**Review by meththot — APPROVED** [review]
Ty

**Comment by meththot** [comment]
Ty

**Comment by palkeo** [comment]
I think it's a bad idea to use sys.argv in some core code directly, like that. You would need to pass it down all the way, but that's also not easy to do…
That said if it takes more than 3 minutes, something is wrong IMO.

**Pull request closed without merging**